<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListView" name="listView">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <layout class="QGridLayout" name="smoothingGrid">
        <item row="0" column="1">
         <widget class="QComboBox" name="smoothingValueCBox">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>No smoothing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Furrier filter</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="smoothingLabel">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Smoothing</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QFrame" name="furrierFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="furrierSmoothingGrid">
           <item row="1" column="0">
            <widget class="QPushButton" name="runSmoothingBtn">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Run smoothing</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="smoothingPerc">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="locale">
              <locale language="English" country="UnitedStates"/>
             </property>
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>95.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="smoothingPercLabel">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Smoothing percentile</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QGridLayout" name="furrierFilterGrid"/>
      </item>
      <item row="6" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="singlePlotTab">
         <attribute name="title">
          <string>Line</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="4" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QFrame" name="figureCanvas">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>5</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>400</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="plotOrdinationTab">
         <attribute name="title">
          <string>Line ordination</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QGridLayout" name="gridLayout"/>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="scanningTab">
         <attribute name="title">
          <string>Scanning</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="loadSpacialCoordinatesBtn">
            <property name="text">
             <string>Load spacial coordinates</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QFrame" name="heatmapQFrame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_folder"/>
    <addaction name="actionLoad_file"/>
    <addaction name="separator"/>
    <addaction name="actionSave_figure"/>
    <addaction name="actionSave_data"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPocessing">
    <property name="title">
     <string>Pocessing</string>
    </property>
    <widget class="QMenu" name="menuBaseline_correction">
     <property name="title">
      <string>Baseline correction</string>
     </property>
     <addaction name="actionarPLS"/>
    </widget>
    <widget class="QMenu" name="menuSmoothing">
     <property name="title">
      <string>Smoothing</string>
     </property>
     <addaction name="actionFurrierFilter"/>
     <addaction name="actionWienerFilter"/>
    </widget>
    <addaction name="menuBaseline_correction"/>
    <addaction name="menuSmoothing"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPocessing"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_folder">
   <property name="text">
    <string>Load folder</string>
   </property>
  </action>
  <action name="actionSave_figure">
   <property name="text">
    <string>Save current figure</string>
   </property>
  </action>
  <action name="actionSave_data">
   <property name="text">
    <string>Save current data</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPolynomial">
   <property name="text">
    <string>Polynomial</string>
   </property>
  </action>
  <action name="actionarPLS">
   <property name="text">
    <string>arPLS</string>
   </property>
  </action>
  <action name="actionFurrierFilter">
   <property name="text">
    <string>Furrirer filter</string>
   </property>
  </action>
  <action name="actionLoad_file">
   <property name="text">
    <string>Load file</string>
   </property>
  </action>
  <action name="actionWienerFilter">
   <property name="text">
    <string>Wiener filter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
